pkg/utils/utils.go

```diff
@@ -209,3 +211,45 @@ func CleanCachedNetConf(cRefPath string) error {
 	}
 	return nil
 }
+
```

Comment:
did you check if we have the method below in sriovnet library? Anyway I think you should move them to it

Reply:
I don't see this changes